.gitignore

```diff
@@ -1,4 +1,6 @@
-# Ignoring Unity meta files since this is a code only plugin
-*.meta
 .DS_Store
 .gradle
+CONTRIBUTING.md.meta
+LICENSE.meta
+README.md.meta
+build.gradle.meta
```

Comment:
We really only need meta files for objects we reference in scenes. Should we keep `*.meta` but add `!GooglePlayInstant/Samples/**/*.meta`?